tests/nexus_test.py

```diff
@@ -206,9 +206,23 @@ def expected_sample() -> sc.DataGroup:
 
 
 @pytest.mark.parametrize('entry_name', [None, nexus.NeXusEntryName('entry-001')])
-def test_load_detector(nexus_file, expected_bank12, entry_name):
+@pytest.mark.parametrize(
+    'data_slice',
+    [
+        None,
+        (
+            'event_time_zero',
+            slice(
+                sc.datetime('1970-01-01T00:00:00'), sc.datetime('2021-01-10T14:16:15')
+            ),
+        ),
+    ],
+)
+def test_load_detector(nexus_file, expected_bank12, entry_name, data_slice):
+    data_slice = {'data_slice': data_slice} if data_slice is not None else {}
     detector = nexus.load_detector(
         nexus_file,
+        **data_slice,
         detector_name=nexus.NeXusDetectorName('bank12'),
         entry_name=entry_name,
     )
```

Comment:
I think a good way to check the loaded data would be to slice during load, and after, and check that the results are the same?

Reply:
Nah this doesn't actually work here. The results are binned by pixel and time and then we can't extract the underlying event coordinates like `event_time_offset`. But when loading from scipp-nexus we can slice by `event_time_offset`.

Reply:
Why not? Use something like https://scipp.github.io/user-guide/binned-data/filtering.html#Extract-events-matching-parameter-value maybe?